src/main/java/com/connexta/util/StringUtils.java

```diff
@@ -0,0 +1,11 @@
+package com.connexta.util;
+
+public class StringUtils {
```

Comment:
❓  Would an alternative option here, since commons-lang and guava are bundles, be to deploy ddf-jsonrpc as a feature and ensure those bundles are available in the container? One side-effect of using a feature would be having to make sure ddf-jsonrpc uses the same versions as DDF to avoid starting up 2 different versions if we care about that.

Reply:
Thats why I went with effectively vendoring in the code we need. No hidden dependencies, gotchas, or version incompatibilities. Most of this code is really simple anyways.